Core/fft_mt_r2iq.cpp

```diff
@@ -188,10 +194,20 @@ void fft_mt_r2iq::Init(float gain, int16_t **buffers, float** obuffers)
 
 		filterplan_t2f_c2c = fftwf_plan_dft_1d(halfFft, pfilterht, filterHw[0], FFTW_FORWARD, FFTW_MEASURE);
 		float *pht = new float[halfFft / 4 + 1];
-		for (int d = 0; d < NDECIDX; d++)
+		for (int d = 0; d < NDECIDX; d++)	// @todo when increasing NDECIDX
```

Comment:
what does it mean about todo? I have another PR to increase NDECIDX

Reply:
> what does it mean about todo? I have another PR to increase NDECIDX

just wanted to note, that here (in the switch below) there's some work necessary (todo), when increasing NDECIDX. wanted to have something visible when looking for all occurences of NDECIDX.

Reply:
i see. i have the change. will submit soon.